### DISCUSSION TITLE
vue中用html的方式引入Cesium.js,页面加载失败。

### DISCUSSION THREAD

**Discussion by HHANG52** [zouyaoji/vue-cesium#459] [Q&A] [answered]
在项目是vue+vite，由于需要离线，就在index.html中用<script src='cesium/Cesium.js'></script> 的方式引入的js。main.js 没写cesiumPath。
页面非正常显示，viewer画面很小/占左上角，别的地方空白，且页面一直在无限往下滚动扩大。

**Comment by zouyaoji** [comment, marked as answer]
这种方式需要你自己把cesium的css也一起引了哈
